Python/sysmodule.c

```diff
@@ -1994,6 +1994,84 @@ sys_getandroidapilevel_impl(PyObject *module)
 }
 #endif   /* ANDROID_API_LEVEL */
 
+/*[clinic input]
+sys.activate_stack_trampoline
+
+    backend: str = "perf"
```

Comment:
I would prefer to make this a mandatory argument. IMO a platform-specific default argument does not make sense.

```suggestion
    backend: str
```

Reply:
:+1: